src/brasil/gov/agenda/browser/agenda.py

```diff
@@ -37,15 +38,20 @@ def agenda_recente(self):
         """Deve retornar a agendadiaria para o dia atual
            caso contrario exibimos
         """
-        agenda = None
-        hoje = DateTime().strftime(AGENDADIARIAFMT)
-        # Validamos se existe uma agenda para o dia de hoje
-        # e se ela esta publicada
-        if hoje in self.context.objectIds():
-            agenda = self.context[hoje]
-            review_state = self.workflow.getInfoFor(agenda, 'review_state')
-            agenda = agenda if review_state == 'published' else None
-        return agenda
+        brains = api.content.find(
+            context=self.context,
+            object_provides=IAgendaDiaria,
+            effective={
+                'query': (DateTime() - 365, DateTime() + 0.1),
+                'range': 'min:max',
+            },
+            sort_on='effective',
```

Comment:
I think it would be easier to `sort_on='Title'` to find the most recent published day.

Reply:
``sort_on='Title'`` is getting wrong day here, but ``sort_on='id'`` works fine

Reply:
I can use ``sort_on='id'`` or ``sort_on='start'``, but still makes sense to me to use the ``max`` range on ``start`` or ``effective`` index because makes sense to have some appointments to the future.